Packs/QRadar/ReleaseNotes/2_0_2.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### IBM QRadar v2
+- Fixed an issue where searching more than 10K indicators failed when using ElasticSearch.
```

Comment:
```suggestion
- Fixed an issue where searching more than 10K indicators failed when using ElasticSearch.
- Updated the Docker image to: *demisto/python3:3.9.4.18682*.
```

Reply:
currently not relevant - I bump it to 2_0_3 due to master changes.